### PR TITLE
chore: remove browserstack

### PR DESCRIPTION
**Pull request by robertsevernsentient**
no value to it
Update the e2e nightly tests to use a headless browser

### CODE REVIEW

File: e2e/launcher/browsers.js [hunk 1]
```diff
@@ -7,16 +7,5 @@ export const browsersForLocal = [
 ];
 
 export const browsersForCI = [
-	'chrome:headless',
-	'firefox:headless'
```
Comment (DT-1236) [inline e2e/launcher/browsers.js:11]: Do we no longer want to run firefox and/or edge?

File: e2e/launcher/run-tests.js [hunk 2]
```diff
@@ -1,9 +1,8 @@
-import { browsersForBrowserStack, browsersForLocal } from './browsers.js';
+import {browsersForCI, browsersForLocal} from './browsers.js';
 import testcafe from "testcafe";
 
 
 export const Mode = {
-	BrowserStack: 'browserstack',
```
Comment (DT-1236) [inline e2e/launcher/run-tests.js:6]: Are we no longer using BrowserStack at all?

File: e2e/launcher/browsers.js (hunk 1 above)
Comment (robertsevernsentient) [reply e2e/launcher/browsers.js:11]: 'firefox:headless' doesn't exist

File: e2e/launcher/run-tests.js (hunk 2 above)
Comment (robertsevernsentient) [reply e2e/launcher/run-tests.js:6]: Correct

**Review by joshua-barnard-evolv — APPROVED** [review]

**Pull request merged**